.gitignore

```diff
@@ -1,5 +1,6 @@
 node_modules
 bower_components
 example/*
+.idea
```

Comment:
This looks to be something specific to your environment and should probably go in your global .gitignore file and not in the project specific one.

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore